dwdiag.bat

```diff
@@ -3,7 +3,7 @@ set "params=%*"
 cd /d "%~dp0" && ( if exist "%temp%\getadmin.vbs" del "%temp%\getadmin.vbs" ) && fsutil dirty query %systemdrive% 1>nul 2>nul || (  echo Set UAC = CreateObject^("Shell.Application"^) : UAC.ShellExecute "cmd.exe", "/k cd ""%~sdp0"" && %~s0 %params%", "", "runas", 1 >> "%temp%\getadmin.vbs" && "%temp%\getadmin.vbs" && exit /B )
 
 IF [%1] == [] (
-	SET branch=master
+	SET branch=kbase

```

Comment:
Why did we not add a parameter for that at launch time ? I'd really like to preserve master as the default branch.

Reply:
👍